content/en/docs/reference/config/analysis/ist0107/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: MisplacedAnnotation
+layout: analysis-message
+---
+
+This message occurs when an Istio annotation is attached to an invalid resource,
+or to a resource in the wrong location.
+
+{{< tip >}}
+Annotation refers to a Kubernetes annotation attached to a pod. For
```

Comment:
How about we define the word annotation in the glossary, and in that glossary entry put a link to the appropriate k8s doc page?

Reply:
fixed